src/View/Widget/RadioWidget.php

```diff
@@ -32,20 +41,49 @@ public function render(array $data, ContextInterface $context)
             'inline' => false,
         ];
 
+        $templates = $this->_templates->get();
+        $customize = [];
+
         if ($data['inline']) {
-            $this->_templates->add(['radioContainer' => '{{content}}']);
+            $this->_inline = $data['inline'];
+            if ($templates['formGroup'] === '{{label}}{{input}}') {
```

Comment:
Using string matching is flaky. It will break if someone for example adds a space in between label and input.